script/bootstrap.py

```diff
@@ -92,6 +95,18 @@ def update_node_modules(dirname, env=None):
       execute_stdout([NPM, 'install'], env)
 
 
+def update_crashpad():
+  gclient = os.path.join(VENDOR_DIR, 'depot_tools', 'gclient.py')
```

Comment:
It seems the only required dependency is `mini_chromium`, I think we should patch crashpad to remove this dependency and use the headers shipped by libchromiumcontent instead, otherwise we would link two versions of `base` library together.